src/containers/ProfilePage/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Container, Row, Accordion } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import UserDetailForm from "../../components/profile/UserDetailForm";
+import UserEmailForm from "../../components/profile/UserEmailForm";
+import UserPasswordForm from "../../components/profile/UserPasswordForm";
+import defaultProfileImage from "../../images/defaultProfileImage.png";
+import ProfileSection from "./ProfileSection";
+import "./index.scss";
+
+export default function ProfilePage({ submit }) {
+  const { register, handleSubmit, errors } = useForm();
+
+  const SECTIONS = [
+    {
+      label: "User details",
+      component: <UserDetailForm register={register} errors={errors} />,
```

Comment:
this should no longer be called component, it should be called children

Reply:
I named it `children` but then changed it to component :smile: